lib/backend/model/workloadendpoint.go

```diff
@@ -150,6 +150,7 @@ type WorkloadEndpoint struct {
 	// TODO: Validation for workload endpoint.
 	State       string            `json:"state"`
 	Name        string            `json:"name"`
+	ContainerID string            `json:"containerid"`
```

Comment:
Lets use one name or the other - it should be trivial to rename this in PyCharm.